### PR TITLE
Union duplicate users from UI

### LINKED ISSUE #101: Union duplicate users
- [x] #126 
- [x] #242 
- [ ] #133

### LINKED ISSUE #133: Persist unified users in `truckconfig.json` from UI

### PR DESCRIPTION
**Pull request by joglr**
Fixes #133 
Fixes #101 

and a few other tweaks

### CODE REVIEW

File: src/components/UnionAuthorsModal.tsx [hunk 1]
```diff
@@ -227,16 +227,13 @@ function CheckboxWithLabel({
...
-  cursor: pointer;
   & > * {
     cursor: initial;
   }
   position: fixed;
   inset: 0;
   padding: calc(4 * var(--unit));
   background-color: hsla(0, 0%, 0%, 0.5);
-  /* Blur background */
-  /* backdrop-filter: blur(var(--unit)); */
```
Comment (joglr) [inline src/components/UnionAuthorsModal.tsx]: rip blured background

**Review by tjomson — APPROVED** [review]
🚀🚀

File: src/components/UnionAuthorsModal.tsx [hunk 2]
```diff
@@ -227,16 +227,13 @@ function CheckboxWithLabel({
 const stringSorter = (a: string, b: string) => a.toLowerCase().localeCompare(b.toLowerCase())
 
 const ModalWrapper = styled.div`
-  cursor: pointer;
   & > * {
     cursor: initial;
```
Comment (joglr) [inline src/components/UnionAuthorsModal.tsx:231]: This should also be removed

Comment (tjomson) [reply src/components/UnionAuthorsModal.tsx:231]: Already is

Comment (joglr) [reply src/components/UnionAuthorsModal.tsx:231]: I mean the & > * { cursor: initial; }

**Pull request merged**